### DISCUSSION TITLE
Question About Concurrency Limits for Free Plan

### DISCUSSION THREAD

**Discussion by Amyhds** [open-meteo/open-meteo#1150] [Q&A] [answered]
Hi,
I'm interested in using your API and would like to know the concurrency limits for the free plan. Specifically, how many simultaneous API requests are allowed?
Thanks in advance!

**Comment by patrick-zippenfenig** [comment, marked as answer]
Hi,
there are no fixed limits for concurrency, but you will be limited after 600 calls per minute: `Less than 10'000 API calls per day, 5'000 per hour and 600 per minute.` https://open-meteo.com/en/terms

However, please keep concurrency to a reasonable limit (<4). Thanks!
